devices/milightRF24.coffee

```diff
@@ -0,0 +1,210 @@
+  _ = require 'lodash'
+  Color = require 'color'
+  nodeMilightRF24 = require 'node-milight-rf24'
+  Buttons = nodeMilightRF24.RGBWButtons
+  nodeMilightRF24 =  nodeMilightRF24.MilightRF24Controller
+  BaseLedLight = require('./base')(env)
+  
+  class MilightRF24
+    constructor: (@config) ->
+      @gateway = new nodeMilightRF24({port: @config.MilightRF24Port})
+      env.logger.debug "Opening "+@config.MilightRF24Port
+      @gateway.open()
+
+      @gateway.on("Received", (data) ->
+        env.logger.debug data
+      )
+    
+    getGateway: ->
+      @gateway
+    
+    getDevice: (config, lastState) ->
+      new MilightRF24Zone(config, lastState, @)
+      
+    setColor: (id, zone, r,g,b) ->
+      env.logger.debug "Sending Color. Addr:"+id+" Zone:"+zone+" Red:"+r+" Green:"+g+" Blue:"+b
+      @gateway.setColor(id, zone, r,g,b)
+      
+    setBrightness: (id, zone, brightness) ->
+      env.logger.debug "Sending Brightness. Addr:"+id+" Zone:"+zone+" Brightness:"+brightness
+      @gateway.setBrightness(id, zone, brightness)
+      
+    setWhite: (id, zone) ->
+      env.logger.debug "Sending Whitemode. Addr:"+id+" Zone:"+zone
+      switch zone
+        when 0
+          button = Buttons.AllOn
+        when 1
+          button = Buttons.Group1On
+        when 2
+          button = Buttons.Group2On
+        when 3
+          button = Buttons.Group3On
+        when 4
+          button = Buttons.Group4On
+      
+      @gateway.sendButton(id, zone, button, true)
+      
+    turnOn: (id, zone) ->
+      env.logger.debug "Sending On. Addr:"+id+" Zone:"+zone
+      switch zone
+        when 0
+          button = Buttons.AllOn
+        when 1
+          button = Buttons.Group1On
+        when 2
+          button = Buttons.Group2On
+        when 3
+          button = Buttons.Group3On
+        when 4
+          button = Buttons.Group4On
+      
+      @gateway.sendButton(id, zone, button, false)
+      
+    turnOff: (id, zone) ->
+      env.logger.debug "Sending Off. Addr:"+id+" Zone:"+zone
+      switch zone
+        when 0
+          button = Buttons.AllOff
+        when 1
+          button = Buttons.Group1Off
+        when 2
+          button = Buttons.Group2Off
+        when 3
+          button = Buttons.Group3Off
+        when 4
+          button = Buttons.Group4Off
+      
+      @gateway.sendButton(id, zone, button, false)
+    
+  class MilightRF24Zone extends BaseLedLight
+
+    constructor: (@config, lastState, MilightRF24Gateway) ->
+      self = @
+      @device = @
+      @gateway = MilightRF24Gateway
+      @zones = @config.zones
+      @brightness = 100
+      @color = "FFFF00"
+      
+      initState = _.clone lastState
+      for key, value of lastState
+        initState[key] = value.value
+      super(initState)
+      if @power then @turnOn() else @turnOff()
+      
+      @gateway.getGateway().on('Received', (data) ->
+        for z in self.zones
+          do br = (z) =>
+            unless z.receive is false
```

Comment:
No it doesn't means the same.
unless z.receive is false = if (z.receive !== false) 
So even if it's not defined it would be true except you put "receive: false" into the config